packages/core/src/components/Dropdown/Dropdown.jsx

```diff
@@ -19,6 +19,18 @@ export class Dropdown extends React.PureComponent {
           `Cannot use 'options' and 'children' React properties at the same time in the <Dropdown> component. Please use 'children' for custom options and 'options' for general cases`
         );
       }
+      // 'ariaLabel' is provided with a `label` prop that is not an empty string
```

Comment:
@scitech @1Copenut lmk if these checks work! I tried to add basic logs for when both `ariaLabel` and `label` are used, and when an empty `label` is used and an `ariaLabel` is necessary.

The checks are a little annoying because `label` is Proptypes.node, so it can be a string or html. There's still the chance that someone can put in `label={<div/>}` or something, and then the Dropdown won't have either `ariaLabel` or a correct `label`, but I'm assuming thats not common.

Reply:
An `axe` check should pick up on improper labels, if they're missing the `htmlFor` attribute. Visual checks would hopefully spot HTML as string output.